src/emulator/auth/state.ts

```diff
@@ -669,6 +705,26 @@ export class TenantProjectState extends ProjectState {
     return this._tenantConfig;
   }
 
+  get allowPasswordSignup() {
```

Comment:
Discussed offline during emulator sync - all these should be default enabled. Will include in next revision